x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -658,7 +635,7 @@ export class VectorStyle extends AbstractStyle {
       return new StaticOrientationProperty(descriptor.options, styleName);
     } else if (descriptor.type === DynamicStyleProperty.type) {
       const field = this._makeField(descriptor.options.field);
-      return new DynamicOrientationProperty(descriptor.options, styleName, field);
+      return new DynamicOrientationProperty(descriptor.options, styleName, field, this._layer);
```

Comment:
Need to update DynamicOrientationProperty to handle new constructor arguments.

Reply:
Its actually `DynamicSizeProperty` constructor that needs to be updated to reflect the changes in signature - https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_size_property.js#L46